src/views/tutorial-library/components/current-filters/current-filters.module.css

```diff
@@ -0,0 +1,31 @@
+.root {
+	display: flex;
+	align-items: center;
+	gap: 8px;
+	margin-bottom: 20px;
+	min-height: 30px;
+
+	& .label {
+		flex-shrink: 0;
+	}
+
+	& ul {
+		list-style-type: none;
+		padding: 0;
+		margin: 0;
+		display: flex;
+		flex-wrap: wrap;
+		gap: 14px 8px;
+
+		& li {
+			display: inline-block;
```

Comment:
```suggestion
			display: flex;
```
It's subtle but the `<li>`s aren't vertically centered. This fixes it on desktop but didn't do much other testing. 
<img width="347" alt="Screen Shot 2022-09-20 at 2 56 34 PM" src="https://user-images.githubusercontent.com/36613477/191372026-79cc77e4-f55c-4a7e-bca8-29fd7a62837e.png">

Reply:
Good catch ty ty ty

Reply:
Unless of course that was intended in the design, looks a bit weird to me though.

Reply:
https://design-system-components-hashicorp.vercel.app/components/tag

Reply:
They do look odd, let's make em as centered looking as possible.